vagrant/provisioning/roles/saml-config/tasks/main.yml

```diff
@@ -52,20 +52,21 @@
     replace: "{{ item.replace }}"
   loop:
     - regexp: 'arkcase/acm/private/saml-keystore'
-      replace: 'arkcase/acm/private/keystore'
+      replace: 'arkcase/acm/private/arkcase.ks'
```

Comment:
it transpires we don't need a special key store for SAML, the default ArkCase key store is fine